specifyweb/frontend/js_src/lib/components/Header/Notifications.tsx

```diff
@@ -72,25 +83,35 @@ export function Notifications({
               }
             >
           >(
-            `/notifications/messages/${queryString}`,
+            formatUrl(`/notifications/messages/`, { queryString }),
             // eslint-disable-next-line @typescript-eslint/naming-convention
-            { headers: { Accept: 'application/json' } },
-            /*
-             * Don't show modal error dialog on errors. Several reasons why:
-             *  - Request may fail if Django migrations weren't run
-             *  - Request is initialized automatically, not by user, thus having
-             *    an error dialog appear out of blue could be confusing/unexpected
-             *  - Notifications is not a critical component, so if it fails, it
-             *    shouldn't bring down entire application
-             */
-            { strict: false }
+            {
+              headers: { Accept: 'application/json' },
+              /*
+               * Don't show modal error dialog on errors. Several reasons why:
+               *  - Request may fail if Django migrations weren't run
+               *  - Request is initialized automatically, not by user, thus having
+               *    an error dialog appear out of blue could be confusing/unexpected
+               *  - Notifications is not a critical component, so if it fails, it
+               *    shouldn't bring down entire application
+               */
+              errorMode: 'silent',
+            }
           )
         )
-        .then(({ data: notifications }) => {
+        .then(({ data: newNotifications }) => {
           if (destructorCalled) return undefined;
-          setNotifications(
-            notifications
-              .map(
+          setNotifications((existingNotifications) => {
+            const filteredNewNotifications = newNotifications.filter(
+              (newNotif) =>
+                !existingNotifications?.some(
+                  (existingNotif) =>
```

Comment:
don't use `Notif` 🙁. doesn't your editor highlight that as a spelling error?
use `Notification` instead